README.md

```diff
@@ -108,9 +108,20 @@ bundle install
 
 To start you can open your terminal and in the folder write
 ```bash
-ruby main.rb
+rails server
 ```
+to drop the existing DB run
+run 
+```bash
+rails db:drop
+```
+Run 
+```
+rails db:create
+```
+to create a new DB.
 
+rails db:migrate to migrate generated db
 
```

Comment:
# Optional: 
- I would advise you to add your partner under the `Authors` section of the readme. Your readme is the first contact between a visitor and your project. Also, Recruiters are more interested in people who could work in a Team or collaboration, therefore adding your partner's data in this template is a big plus.
- NB: This comment is not in line with the code because you did not make the code visible